libkernelflinger/trusty_efi.c

```diff
@@ -58,6 +57,13 @@
 #define BASE_4GB                 0x100000000ULL
 #endif
 
+#define RPMB_KEY_SIZE    32
+#define RPMB_SEED_SIZE  32
+#define RPMB_NUMBER_KEY  10
+#define MMC_PROD_NAME_WITH_PSN_LEN   15
+#define RPMB_MAX_PARTITION_NUMBER 6
+#define RPMB_MAX_KEY_SIZE 64
+
```

Comment:
are they used in kernelflinger?

Reply:
These are required by "struct tos_startup_info" which is the interface between kernelflinger and evmm.  I keep the interface in this patch to avoid boot failure issue. 
The next patch will only keed seed[64] and remove rpmb in "struct tos_startup_info". The related patch in ikgt has already submitted. 
https://github.com/intel/ikgt-core/pull/108